test/sagemaker_tests/huggingface_pytorch/training/integration/sagemaker/test_smmp.py

```diff
@@ -26,8 +26,8 @@
 hyperparameters = {
     "model_name_or_path": "hf-internal-testing/tiny-random-RobertaModel",
     "task_name": "mnli",
-    "per_device_train_batch_size": 1,
-    "per_device_eval_batch_size": 1,
+    "per_device_train_batch_size": 2,  # batch sizes must be divisible by microbatche size
```

Comment:
Not fully following this - what isn't divisible by 1?

Reply:
@mabunday  Without this change, we get below error. Note that the microbatch size is set to 2. And `2 % 1 = 1 != 0`. 


```
[1,mpirank:2,algo-1]<stderr>:TensorSplitError: Batch size must be divisible by the number of microbatches. 
[1,mpirank:2,algo-1]<stderr>:Found batch size=1, number of microbatches=2. If this is the last batch of the 
```

Reply:
https://github.com/aws/deep-learning-containers/blob/e1adf2c467c7bd1faf43fc867de233b91fc947d5/test/sagemaker_tests/huggingface_pytorch/training/integration/sagemaker/test_smmp.py#L47

Reply:
nit: Can we define a variable and set it so they're the same (or derive one from the other) then if there's this implicit dependency?

Reply:
Hmm sorry not sure if I get your point - could you elaborate?